### ISSUE TITLE
Very slow behaviour in native WP Customizer, any ideas? :)

### ISSUE THREAD

**Issue by WPJobs** [ThemeFuse/Unyson#2483]
Hi all!

Unyson Developers, I know that you're not obligated to fix this, so just a question:

Do you know maximum number of settings that i can have in native WP customizer using Unyson settings config? 

Currently i have ~300 options in Customizer and they are really really slow, i need to wait ~1-1.5 sec when i click / navigate between customizer sections

Anybody, do you use native WP customizer? How many settings can you advice to add

Thank you and have a great day all!

**Comment by GheorgheP** [comment]
Yes, you are right, I just tried with about 300 options and get same slow behavior. After my tests I got with something about 100-150 options.

We will try to optimize this.

**Comment by WPJobs** [comment]
Thank you for your reply. If this will help: Chrome has mostly slow behavior, Firefox and opera are much better. Also, freezing happens on backend options if you navigate between tabs and change about 10-30 settings, but it works better than customizer. My idea was to have the same options in backend and customizer sides. Anyway, thank you so much for this great framework, I still use it for everyof my projects just because I'm loving it!

**Comment by WPJobs** [comment]
P.s. Is Moldcraft left Unyson team?

**Comment by danyj** [comment]
@WPJobs https://github.com/ThemeFuse/Unyson/issues/3620
